### PR TITLE
Add wagtail

### PR DESCRIPTION
**Pull request by marcoacierno**

### CODE REVIEW

File: backend/requirements/base.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
-Django==1.11
```
Comment (marcoacierno) [inline backend/requirements/base.txt]: The version of wagtail I used when I started the integration wasn't compatible with django 1.11 yet

File: backend/requirements/base.txt [hunk 2]
```diff
@@ -2,3 +2,4 @@ Django==1.11.3
 wagtail==1.11.1
```
Comment (marcoacierno) [inline backend/requirements/base.txt]: why?

File: backend/pycon/settings/base.py [hunk 3]
```diff
@@ -136,7 +136,7 @@
 MEDIA_URL = '/media/'
```
Comment (marcoacierno) [inline backend/pycon/settings/base.py]: done

File: backend/requirements/base.txt (hunk 2 above)
Comment (marcoacierno) [inline backend/requirements/base.txt]: maybe the comment is for 'psycopg2==2.7.1' and not wagtail?

Comment (patrick91) [inline backend/requirements/base.txt]: I'd just use postgres everywhere :)

Comment (simobasso) [inline backend/requirements/base.txt]: 'cause the actually default is `sqlite`.

If a collaborator w/o pg start an instances with that commit, the setup will fail.

options:

1. move `psycopg2==2.7.1` in production' dependencies
2. use pg as default in settings_base.py

Comment (patrick91) [inline backend/requirements/base.txt]: let's use postgres everywhere :)

**Pull request closed without merging**